index.html

```diff
@@ -87,32 +90,32 @@ <h2 class="sub-heading font-heading text-center sub-heading headings-common_styl
+                    <h2 class="card-heading font-heading margin-bottom-small">Custom Software Development</h2>
+                    <p class="paragraph-common margin-bottom-medium card-paragraph paragraph font-body">For those of you that use desktop/laptop computers, if hardware are the body and skeleton of the computer, then software are the instructions that travel from the brain. We all use software to make working with our computers easier, but what if a software doesn't work the way you want? That's where we come in. We have skilled developers who can create any software for you, from scratch. Your input is valuable, as it may help us be more precise and speed up the process.</p>
                 </div>
                 <div class="card lax lax_preset_fadeInOut:233.25:0 lax_preset_blurInOut:233.25:20 lax_preset_scaleInOut:233.25:0.6 padding-left-small padding-right-small margin-top-medium margin-bottom-medium flex-col a-center flex text-center">
                     <lottie-player class="margin-top-small" src="https://assets2.lottiefiles.com/packages/lf20_wloxwco0.json"  background="transparent"  speed="1"  style="width: 60px; height: 60px;"></lottie-player>
-                    <h2 class="card-heading font-heading margin-bottom-small">Mobile Applications Development</h2>
-                    <p class="paragraph-common margin-bottom-medium card-paragraph paragraph font-body">Lorem ipsum dolor sit, amet consectetur adipisicing elit. Rem cupiditate amet aliquam incidunt facere ratione alias recusandae maiores, debitis nisi numquam, adipisci atque. Ipsam illo velit sed at recusandae! Asperiores.</p>
+                    <h2 class="card-heading font-heading margin-bottom-small">Website and Web Application Development</h2>
+                    <p class="paragraph-common margin-bottom-medium card-paragraph paragraph font-body">Unsurprisingly, this is, without a doubt, one of the MOST demanded and used service in the Internet world. Every second of every day a website is accessed in order to aquire information or services, you know, like how you are doing right now. So it stands to reason that this is our most demanded and iconic service in Qodbrains. We especially pride ourselves in being able to make Websites and Web Apps from the beginning, with very little to none outside resources (icons for example). We craft websites with these 3 languages: HTML5 (For the basic structure), CSS3 (For styling like colors), and JavaScript (For making sites more dynamic and interactive). Occasionally the are clients who demand a speedy completion of their website, that's when we use other resources and libraries to speed up our work - like React, Bootstrap, etc.</p>
```

Comment:
change: _We craft websites with these 3 languages: HTML5 (For the basic structure), CSS3 (For styling like colors), and JavaScript (For making sites more dynamic and interactive)...._

to: We craft websites using the latest and most secure tools for the job. 

remove: _Occasionally the are clients who demand a speedy completion of their website, that's when we use other resources and libraries to speed up our work - like React, Bootstrap, etc._